Makefile

```diff
@@ -285,8 +285,8 @@ packagedir: clean check_bundledir $(BUILD_DIR)/macos-amd64/crc $(HOST_BUILD_DIR)
 $(BUILD_DIR)/macos-amd64/crc-macos-amd64.pkg: packagedir
 	./packaging/package.sh $(BUILD_DIR)/macos-amd64
 
-$(BUILD_DIR)/macos-amd64/crc-installer.tar: packagedir
-	tar -cvf $(BUILD_DIR)/macos-amd64/crc-installer.tar ./packaging
+$(BUILD_DIR)/macos-amd64/crc-macos-installer.tar: packagedir
+	tar -cvf $(BUILD_DIR)/macos-amd64/crc-macos-installer.tar ./packaging
```

Comment:
What is the impact for people consuming this tarball? Do they need to adjust some scripts?

Reply:
`tar -cvf $(BUILD_DIR)/macos-amd64/crc-macos-installer.tar ./packaging` could be `tar -cvf $@ ./packaging` I think

Reply:
>What is the impact for people consuming this tarball? Do they need to adjust some scripts?

I don't think anyone except us is consuming this tarball because this is provided to RCM team to sign it.

Reply:
Then the RCM team is consuming the tarball ;) Do they need to adjust their scripts? Can we rename it to `crc-macos-amd64-unsigned.tar` ?

Reply:
> Then the RCM team is consuming the tarball ;) Do they need to adjust their scripts?

@cfergeau  I don't think they consume it as part of script, they first download this artifact using wget and then run what is required.

> Can we rename it to crc-macos-amd64-unsigned.tar ?

This is the tarball not the `.pkg` file which require signed so I am not sure why we have to put `unsigned` on the tarball side?